ui/app/registration/directives/topDownAddressFields.js

```diff
@@ -44,7 +48,20 @@ angular.module('bahmni.registration')
                 });
             }
         };
-
+        $scope.translateAttributes = function (attribute, moduleName) {
+            if (moduleName == null) {
+                var keyPrefix = "REGISTRATION";
+            } else {
+                var keyPrefix = moduleName && modulePrefixMap[moduleName] ? modulePrefixMap[moduleName] : '';
+            }
+            var keyName = attribute.toUpperCase().replace(/\s\s+/g, ' ').replace(/[^a-zA-Z0-9 _]/g, "").trim().replace(/ /g, "_");
+            var translationKey = keyPrefix + "_" + keyName;
+            var translation = $translate.instant(translationKey);
+            if (translation != translationKey) {
+                attribute = translation;
+            }
+            return attribute;
+        };
```

Comment:
@buvaneswari-arun same comment as above

Reply:
@buvaneswari-arun As this method is not private and being used in multiple places, please add test cases for this piece of code

Reply:
Testcases will be taken up as a part of second cut.